envJsonConverter.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Very much your choice, but I've noticed that this is the sort of thing I used to solve with a bash file because it was a "small simple script", but actually when I changed to using python files I found them quicker to write and maintain

Reply:
Yeah this definitely would have been easier in Python and that's probably what I would have done in the real world, but I've not written many bash scripts before so thought it would be worthwhile for the exercise